### DISCUSSION TITLE
Complete First Version Without Responsive.

### DISCUSSION THREAD

**Discussion by MrBns** [MrBns/nex-bridge#10] [General]
Sections for @MrBns 

- [x] Navbar
- [x] Header
- [x] we believe
- [x] why Choose us
- [x] Map Flag Section

-----------------------

Sections for @iamajraj 

- [x] Blogs Section
- [x] want to connect section
- [x] Footer
- [x] brand slider
- [x]  traditional Finance Section

**Comment by MrBns** [comment]
@iamajraj  Mark you section completed..

**Comment by iamajraj** [comment]
Yep

**Comment by MrBns** [comment]
brand slider added.

**Comment by iamajraj** [comment]
- Traditional Finance Section added
